integration-test/src/main/java/com/scalar/db/api/DistributedStorageSingleClusteringKeyScanIntegrationTestBase.java

```diff
@@ -79,8 +79,8 @@ protected String getNamespace() {
     return NAMESPACE;
   }
 
-  protected Set<DataType> getClusteringKeyTypes() {
-    return new HashSet<>(Arrays.asList(DataType.valuesWithoutTimesRelatedTypes()));
+  protected List<DataType> getClusteringKeyTypes() {
```

Comment:
For two reasons: 
1. To uniformize the similar types of methods returning the data types in the storage integration test. Some class were using Set, others were List. 
2. For practical reasons since I added this method https://github.com/scalar-labs/scalardb/blob/dce86f75aa93d94a0134b68394ee6af671aba36c/core/src/integration-test/java/com/scalar/db/storage/jdbc/JdbcTestUtils.java#L76-L102 
that is used like this below to filter out non-testable data types on JDBC integration test and not requiring the method to handle a set or a list was simpler.
https://github.com/scalar-labs/scalardb/blob/dce86f75aa93d94a0134b68394ee6af671aba36c/core/src/integration-test/java/com/scalar/db/storage/jdbc/JdbcDatabaseMultiplePartitionKeyIntegrationTest.java#L80-L91